test/mocha/browserstack-capabilities-test.js

```diff
@@ -3,27 +3,42 @@ var browserStackProvider = require('../../');
 
 
 describe('Browserstack capabilities', function () {
+    after(function () { 
+        delete process.env.BROWSERSTACK_TEST_RUN_NAME;
+        delete process.env.BROWSERSTACK_GEO_LOCATION;
+        delete process.env.BROWSERSTACK_CUSTOM_NETWORK;
+        delete process.env.BROWSERSTACK_NETWORK_PROFILE;
+    });
+
```

Comment:
Why do we need this code? Do these variables affect the further test execution?

Reply:
Yes, they do.

Reply:
You are right, they definitely do.
The `BROWSERSTACK_TEST_RUN_NAME` var goes to the `BrowserStack Dashboard` and `BROWSERSTACK_CUSTOM_NETWORK/BROWSERSTACK_NETWORK_PROFILE` emulates Network condition. 
Could you please explain why you delete the `BROWSERSTACK_GEO_LOCATION` variable? It seems that that var does not affect the further tests.

Reply:
Looks like BrowserStack has a bug when I use the `geoLocation` capability with BrowserStack Automate. If I do not delete the `process.env.BROWSERSTACK_GEO_LOCATION` code, BrowserStack fails for some reason. So, this code is also required to run the tests.

Reply:
Thanks! It looks like this also addresses https://github.com/DevExpress/testcafe-browser-provider-browserstack/issues/49